content/influxdb/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,50 @@ menu:
 weight: 101
 ---
 
+## v2.0.3 General Availability [2020-12-14]
+
+### Breaking Changes
+
+#### `influxd upgrade`
+
+Previously, `influxd upgrade` would attempt to write upgraded `config.toml` files into the same directory as the source
+`influxdb.conf` file. If this failed, a warning would be logged and `config.toml` would write into the `home` directory of the user who launched the upgrade.
+
+This release breaks this behavior in two ways:
+- By default, `config.toml` writes into the same directory as the Bolt DB and engine files (`~/.influxdbv2/`)
+- If writing upgraded config fails, the `upgrade` process exits with an error instead of falling back to the `HOME` directory
+
+To override the default configuration path (`~/.influxdbv2/`), use the new `--v2-config-path` option to specify the output path to the v2 configuration file (`config.toml`). For details, see [Upgrade from InfluxDB 1.x to InfluxDB 2.0](/influxdb/v2.0/upgrade/v1-to-v2/).
+
+#### InfluxDB v2 packaging
+
+We've renamed the InfluxDB v2 DEB and RPM packages to clarify versions. The package name is now `influxdb2` and conflicts with any previous `influxdb` package (including initial 2.0.0, 2.0.1, and 2.0.2 packages). 
+
+This release also defines v2-specific path defaults and provides helper scripts for `influxd upgrade` and cleanup cases.
+
+### Features
+
+- Allow password to be specified as a CLI option in [`influx v1 auth create`](/influxdb/cloud/reference/cli/influx/auth/create/).
+- Allow password to be specified as a CLI option in [`influx v1 auth set-password`](/influxdb/cloud/reference/cli/influx/auth/).
+- Implement [delete with predicate](/influxdb/v2.0/write-data/delete-data/).
+- Improve ID-related error messages for `influx v1 dbrp` commands.
+- Update Flux to v0.98.0.
```

Comment:
to [Flux v0.98.0](/influxdb/v2.0/reference/release-notes/flux/#v0-98-0-2020-12-07).

Reply:
Changed as of this afternoon: we're now at Flux v0.99.0 and flux-lsp-browser v0.5.26